cmd/lakectl/cmd/store/s3.go

```diff
@@ -12,15 +14,38 @@ import (
 	"github.com/aws/aws-sdk-go/service/s3/s3iface"
 )
 
-func GetS3Client() (*s3.S3, error) {
-	sess, err := session.NewSessionWithOptions(session.Options{
-		SharedConfigState: session.SharedConfigEnable,
-	})
-	if err != nil {
-		return nil, err
+func GetS3Client(intg string) (*s3.S3, error) {
+	if intg == "minio" {
+		// Configure to use MinIO Server
+		accountName, accessKey, secretKey := os.Getenv("MINIO_STORAGE_ACCOUNT"), os.Getenv("MINIO_STORAGE_ACCESS_KEY"), os.Getenv("MINIO_STORAGE_SECRET_ACCESS_KEY")
```

Comment:
I cannot find documentation for these environment variables.  There is [no reference](https://www.google.com/search?q=site%3Amin.io+"MINIO_STORAGE_ACCOUNT") to these environment variables in min.io or on [the Minio organization on GitHub](https://github.com/search?q=org%3Aminio+%22MINIO_STORAGE_ACCOUNT%22&type=code).